src/rt/sched/cown.h

```diff
@@ -1038,16 +996,9 @@ namespace verona::rt
     {
       auto until = queue.peek_back();
       yield(); // Reading global state in peek_back().
-
-      const auto stat = status.load(std::memory_order_acquire);
       assert(priority() != Priority::Low);
 
-      // The batch limit is between 100 and 251, depending on the load.
-      const auto batch_limit = (size_t)100 | ((size_t)stat.total_load() >> 3);
-
-      Systematic::cout() << "Cown " << this << " load: " << stat.total_load()
-                         << std::endl;
-
+      static constexpr size_t batch_limit = 100;
```

Comment:
Unused?

Reply:
It's used in the do-while condition down at line 1103.